open-company/open-company-web
# Slack invite

card: https://trello.com/c/LNR4dpA3

What does this branch do:
- add Slack invite grouped by Slack organization
- add team logo to the newly created org

To test:
- signup with Slack
- create a new organization
- check that the new org has the name and the logo of you Slack org
- go to Team Management
- add the bot to your Slack team
- select Slack under Invite and send the invite for one of you Slack coworker
- to check the message signin to that slack account and check the messages from opencompany bot
- add another Slack org to the team
- add the bot to this second Slack team
- open the Slack invite and check that the users are grouped by Slack org
- invite another user
- click resend invite for one of the 2 users you just invited
- the user should receive the invitation by email the second time
- check that the email has the company name and logo (that was the logo of the first Slack org you used to signup)
- merge
